Adding a Baz tile (#2461)
DataDog/integrations-extras: baz/README.md

# Baz

## Overview

> Baz helps developers write unbreakable code by:

Reviewer: I'd suggest changing `unbreakable` to `thoroughly validated`.

Author: It's the brand... https://baz.co

1. Identifying what the code represents, and part of what application / platform it operates in.
2. Identifying dependencies and the possible impact a code change may bring.
3. Bringing all this context into the review process and making it 100x more collaborative and constructive.

## Setup

### Installation

Visit [baz.co][1] and sign up. Once registered, visit the [Datadog integration page][2] and add a Datadog integration. This guides you through the Datadog OAUTH2 flow to grant Baz access to your billing and usage data.

### Configuration

Once integrated, you will see the integration with its status in the [Datadog integration page][2].

### Uninstallation

Visit the [Datadog integration page][2] and click the "Remove" trash button to uninstall the integration.

## Data Collected

> Baz collects APM metrics - traces, spans and services, to get the app context so missing in today's code reading and reviewing.

Reviewer: ```suggestion
Baz collects APM metrics on traces, spans and services, to provide information on your apps during code reviews.
```
